src/app.js

```diff
@@ -16,11 +16,14 @@ const decksRouter = require('./routes/Decks.js');
 const allUnauthDecksRouter = require('./routes/decksAllUnauth');
 
 // middleware
-app.use(cors());
+app.use(cors({
+    origin: 'http://localhost:3000',
```

Comment:
[Configurability / Hardcoded Values]
This should be a configuration value in the .env.
That way if the front-end proxy address is something other than localhost:3000 the app still works.

Reply:
Added the path to the .env file. Thank you